docs/RepoAndNode.md

```diff
@@ -17,3 +17,122 @@ After a repo is created, you can store data in it. The data is stored as `nodes`
 To get, create, modify or delete nodes, a connection to a repo is neccessery. It is done by 
 calling `connect` from enoics node library.
  
+## Dataset
+automatically sets up `no.ssb.dataset` repo and one child node for each of the different dataset sources.
+
+### `getDataset()`
+Example
+```typescript
+const myDataset: DatasetRepoNode<MyDatasetType> = getDataset(DataSource.TALLBYGGER, '123456')
+```
+Returns (undefined if the node with the key doesnt exist)
```

Comment:
Could we clip the result from `get` to show only the relevant fields (leave out XP node metadata)  for readability ?

Reply:
The thought was to do that in the ssb/dataset lib, or somewhere else. So when you're using the repo/dataset lib, you're getting the full node

Reply:
Makes sense. I will go through the ssb::statreg doc and see if I can make the changes in one of my subsequent branches/PRs